addons/website_event_sale/views/templates.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="event_confirmation" inherit_id="website_sale.confirmation">
+        <xpath expr="//div[@id='oe_structure_website_sale_confirmation_2']" position="inside">
+            <t t-if="events">
+                <section class="s_title pt40" data-snippet="s_title" data-name="Title">
+                    <div class="s_allow_columns container">
+                        <h4>
+                            We are looking forward to meeting you at the following
+                            <t t-if="len(events) == 1">event</t>
+                            <t t-else="">events</t>
+                            :
+                        </h4>
+                    </div>
+                </section>
+                <section class="pb32 o_cc o_cc2 o_colored_level bg-transparent">
+                    <div class="container">
+                        <div class="s_nb_column_fixed s_col_no_bgcolor o_wevent_index">
+                            <t t-foreach="events" t-as="event">
```

Comment:
Note that the foreach can be put directly into its sub-div tag, leading to less indentation levels